src/website/src/pages/ec/components/media/media-container/docs/code.mdx

```diff
@@ -28,3 +28,26 @@ import demoContentVideo from '@ecl/ec-specs-media-container/demo/data--video';
 >
   <MediaContainer {...demoContentVideo} />
 </Playground>
+
+## Embedded video (using &lt;iframe&gt;)
+
+Note: for embedded videos, a ratio has to be specified if order to display the iframe in a responsive way.
```

Comment:
Should be: Note: for embedded videos, a ratio has to be specified **_in_** order to display the iframe in a responsive way.